sandbox.sh

```diff
@@ -188,17 +188,21 @@ EOF
     ];
   };
   vault = {
-    known_handles_hash = (Big_map.empty : vault_known_handles_hash_set);
-    used_handles = (Big_map.empty : vault_used_handle_set);
-    vault = (Big_map.empty : vault);
-  }
+    known_handles_hash = (Big_map.empty : Parameter.Types.vault_known_handles_hash_set);
+    used_handles = (Big_map.empty : Parameter.Types.vault_used_handle_set);
+    vault = (Big_map.empty : Parameter.Types.vault);
+  };
+  metadata = Big_map.literal [
+    ("data", 0x7b0a202020226e616d65223a2244656b75207469636b657473222c0a202020226465736372697074696f6e223a2254686520627269646765206265747765656e204c61796572203120616e64204c617965722032222c0a20202022617574686f7273223a5b0a2020202020202244656b7520446576205465616d203c68747470733a2f2f7777772e6d617269676f6c642e6465762f7465616d3e220a2020205d2c0a20202022686f6d6570616765223a2268747470733a2f2f7777772e6d617269676f6c642e6465762f70726f6a6563742f64656b752d73696465636861696e222c0a20202022696e7465726661636573223a5b0a20202020202022545a49502d303132220a2020205d2c0a20202022646563696d616c73223a2230222c0a2020202273796d626f6c223a2244454b54220a7d);
```

Comment:
**Please add a layer1/bridge/metadata.json file and convert to hex here.**

Metadata aren't ok, this is not a FA2 token. I propose the following:
```json
{
   "name":"DEKU",
   "description":"The bridge between Layer 1 and Layer 2",
   "version": "0.1.0",
   "license": {"name": "MIT"},
   "authors":[
      "Marigold <contact@marigold.dev>"
   ],
   "homepage":"https://www.marigold.dev/project/deku-sidechain",
   "source":{
      "tools": ["CameLIGO"],
      "location": "https://github.com/marigold-dev/deku/ree/main/layer1/bridge"
   }
}
```

Reply:
Also there is a project to replace sandbox.sh by an ocaml CLI, so orgination with metadata would wait or rebase on it to avoid double work